### DISCUSSION TITLE
Установка и удаление

### DISCUSSION THREAD

**Discussion by ToTohin** [Anonym-tsk/tpws-keenetic#13] [Обсуждения]
Подскажите, пожалуйста, почему не могу удалить конфиг?
![image](https://github.com/user-attachments/assets/98c36de4-657f-4481-9fc7-4fab2e23f2b5)

**Comment by Anonym-tsk** [comment]
Что-то у вас с днс не так, проверяйте настройки роутера

**Reply by ToTohin** [reply]
Спасибо большое) Снял вот эту галочку и все заработало.
![image](https://github.com/user-attachments/assets/d4b47bd9-a1d8-4c86-ac50-fbeaced1d18e)

**Comment by ToTohin** [comment]
Подскажите, еще, пожалуйста, почему нет доступа к списку доменов?) Что-то делаю не так.
![image](https://github.com/user-attachments/assets/2a9f619b-b79a-459e-97cd-130c48c20fa8)

**Reply by Anonym-tsk** [reply]
Потому что вы пытаетесь выполнить текстовый файл (к тому же, это лог). Его нужно открыть текстовым редактором, например vi

**Comment by SausageKiller218** [comment]
Возможно ли установить на роутер с arm-v7? Archer ax73 w/ entware
Или есть какие-либо другие варианты?

Выбирая aarch64 оно выдаёт ошибки и сервис по итогу не запускается

**Reply by Anonym-tsk** [reply]
Добавил поддержку arm
